Unable to import types

In my typescript project I am having issues importing types such as the BracketsViewer. It appears that this was completed according to https://github.com/Drarig29/brackets-viewer.js/issues/27 but I am using version 1.6.2 and I am unable to import the types.

I am not much of a typescript guru but I believe that either the tsconfig.json needs declaration set to true and/or the package.json needs to have the types property set. I am willing to look into this issue and make a fix if problem is not between chair and keyboard.

For more context I am able to import BracketsManager like so:
```ts
import { BracketsManager } from 'brackets-manager/dist'
```

But I cannot import BracketsViewer, I have tried the following:
```ts
import { BracketsViewer } from 'brackets-viewer/dist';
```
and the following as mentioned in https://github.com/Drarig29/brackets-viewer.js/issues/27
```ts
import { BracketsViewer } from 'brackets-viewer';
```

I get the following error: TS2307: (TS) Cannot find module 'brackets-viewer' or its corresponding type declarations.

--- a/package.json
+++ b/package.json
@@ -2,18 +2,21 @@
     "name": "brackets-viewer",
     "version": "1.6.2",
     "description": "A simple library to display tournament brackets (round-robin, single elimination, double elimination)",
+    "main": "./dist/index.js",
+    "types": "./dist/index.d.ts",
     "files": [
         "dist/"
     ],
     "scripts": {
         "db": "npx json-server --watch demo/db.json",
-        "build": "npx webpack --mode production",
+        "build": "npx webpack --mode production && npm run tsc",
         "start": "npx webpack --watch --mode development",
         "prepare": "npm run build",
         "prepublishOnly": "npm run build",
         "lint": "eslint '**/*.{js,ts}' --fix",
         "test": "exit 0",
-        "watch-demo": "browser-sync demo . --watch dist"
+        "watch-demo": "browser-sync demo . --watch dist",
+        "tsc": "tsc --emitDeclarationOnly"
     },
     "repository": {
         "type": "git",
@@ -65,6 +68,6 @@
         "ts-loader": "^9.2.8",
         "typescript": "^4.6.3",
         "webpack": "^5.70.0",
-        "webpack-cli": "^4.9.2"
+        "webpack-cli": "^4.10.0"
     }
 }

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -4,6 +4,8 @@
         "module": "CommonJS",
         "moduleResolution": "node",
         "strict": true,
+        "declaration": true,
+        "outDir": "./dist",
         "types": [
             "brackets-model"
         ],

